metadata/src/main/java/org/apache/kafka/metadata/bootstrap/BootstrapMetadata.java

```diff
@@ -73,7 +73,7 @@ public static Optional<MetadataVersion> recordToMetadataVersion(ApiMessage recor
         return Optional.empty();
     }
 
-    BootstrapMetadata(
+    public BootstrapMetadata(
```

Comment:
This should not be public. Use `public static BootstrapMetadata fromRecords(List<ApiMessageAndVersion> records, String source)` instead.

Reply:
Fixed